modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/view_kb_article.jsp

```diff
@@ -286,6 +286,8 @@ String kbArticleSuccessMessage = GetterUtil.getString(MultiSessionMessages.get(r
 		module="{LockedKBArticleModal} from knowledge-base-web"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
+				"isAdmin", permissionChecker.isGroupAdmin(scopeGroupId)
```

Comment:
due consistency "isAdmin" shouldn't be "isGroupAdmin" or directly "groupAdmin"?

Reply:
I agree with Roberto, "groupAdmin" seems more accurate. At least in Java (and I think this applies to Js as well) the convention for boolean variables is to drop the "is"; e.g. `boolean foo = object.isFoo()`.

Reply:
Change done here [348c04f](https://github.com/liferay-content-management/liferay-portal/pull/5187/commits/348c04fe311769916872a826645f5fe4bb57e0e3)